docker/php/docker-entrypoint.sh

```diff
@@ -7,11 +7,9 @@ if [ "${1#-}" != "$1" ]; then
 fi
 
 if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
-	PHP_INI_RECOMMENDED="$PHP_INI_DIR/php.ini-production"
 	if [ "$APP_ENV" != 'prod' ]; then
-		PHP_INI_RECOMMENDED="$PHP_INI_DIR/php.ini-development"
+		ln -sf "$PHP_INI_RECOMMENDED" "$PHP_INI_DIR/php.ini-development"
```

Comment:
Shouldn't it be more like this?
```suggestion
		ln -sf "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini"
```

Reply:
Ooops ^^